examples/connectivity/v4/port2serviceprofilepublic/terraform.tfvars

```diff
@@ -10,7 +10,7 @@ redundancy = "PRIMARY"
 purchase_order_number = "1-323292"
 aside_ap_type = "COLO"
 aside_port_uuid = "c4d9350e-783c-83cd-1ce0-306a5c00a600"
-aside_link_protocol_type = "QINQ"
+aside_link_protocol_type = "DOT1Q"
 aside_link_protocol_stag = "2019"
```

Comment:
Similar to other comments about variable matching context where it will be used.

Reply:
If the zSide is the same in `main.tf` used just `_tag`, then that should be updated as well.